### PR TITLE
add multiple location markers while using the marker click listener

### PR DESCRIPTION
**Pull request by leeannachen**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -287,18 +288,51 @@ var styledMapType = new google.maps.StyledMapType(
...
     });
 
-    //Associate the styled map with the MapTypeId and set it to display.
+    // Associate the styled map with the MapTypeId and set it to display.
     map.mapTypes.set('styled_map', styledMapType);
     map.setMapTypeId('styled_map');
+
+    // Create landmark
+    addLandmark(
+        map, 36.164347196601156, -86.78117037227406, 'Nashville',
```
Comment (aokeke) [inline portfolio/src/main/webapp/script.js]: Maybe get these numbers from Nashville.lat, Nashville.lng

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -287,18 +288,51 @@ var styledMapType = new google.maps.StyledMapType(
...
+    // The location of Nashville
+    var Nashville = {lat: 36.164347196601156, lng: -86.78117037227406};
+
     // Create a map object, and include the MapTypeId to add
     // to the map type control.
+    // The map, centered at Nashville
     var map = new google.maps.Map(document.getElementById('map'), {
-        center: {lat: 36.164347196601156, lng: -86.78117037227406},
-        zoom: 14,
+        center: {lat: 36.164347196601156, lng: -80.78117037227406},
```
Comment (aokeke) [inline portfolio/src/main/webapp/script.js]: Is this supposed to change without changing the value above for Nashville? Can you just plug in Nashville here?

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -287,18 +288,51 @@ var styledMapType = new google.maps.StyledMapType(
     ],
             {name: 'Styled Map'});
 
+    // The location of Nashville
+    var Nashville = {lat: 36.164347196601156, lng: -86.78117037227406};
```
Comment (aokeke) [inline portfolio/src/main/webapp/script.js]: I think JS conventions say this should be lowercase n `nashville`

**Review by aokeke — CHANGES_REQUESTED** [review]
I like the compactness of this PR.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (leeannachen) [reply portfolio/src/main/webapp/script.js]: I changed the view of the map along with its center so it is not zoomed in on Nashville but instead shows a view of the eastern US for narrative purposes.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (leeannachen) [reply portfolio/src/main/webapp/script.js]: I will try that

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (leeannachen) [reply portfolio/src/main/webapp/script.js]: Got it

**Review by leeannachen — COMMENTED** [review]
update map using feedback and changed zoom view

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -287,18 +288,51 @@ var styledMapType = new google.maps.StyledMapType(
...
+    // The location of Nashville
+    var nashville = {lat: 36.164347196601156, lng: -86.78117037227406};
+
     // Create a map object, and include the MapTypeId to add
     // to the map type control.
+    // The map, centered at Nashville
     var map = new google.maps.Map(document.getElementById('map'), {
-        center: {lat: 36.164347196601156, lng: -86.78117037227406},
-        zoom: 14,
+        center: {lat: 38, lng: -88},
```
Comment (aokeke) [inline portfolio/src/main/webapp/script.js:299]: I'm still wondering why this isn't `Math.round(nashville.lat)` and `Math.round(nashville.lng)`, but this still looks good nonetheless. I'm assuming you've run it and it centers the map the way you like it.

**Review by aokeke — APPROVED** [review]

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (leeannachen) [reply portfolio/src/main/webapp/script.js:299]: yup, the center is adjusted to also show location markers in CO, CT, and MA

**Pull request merged**